apps/web/lib/QueryCell.tsx

```diff
@@ -72,3 +80,25 @@ export function QueryCell<TData, TError extends ErrorLike>(
   // impossible state
   return null;
 }
+
+type TQueryValues = inferProcedures<AppRouter["_def"]["queries"]>;
+type TQueries = AppRouter["_def"]["queries"];
+type TError = TRPCClientErrorLike<AppRouter>;
+
+const withQuery = <TPath extends keyof TQueryValues & string>(
+  pathAndInput: [path: TPath, ...args: inferHandlerInput<TQueries[TPath]>],
```

Comment:
Type error: `Type 'Prefixer<Record<"session", Procedure<{ i18n: SSRConfig; prisma: PrismaClient<PrismaClientOptions, never, RejectOnNotFound | RejectPerOperation | undefined>; session: Session | null; user: { ...; } | null; locale: string; }, ... 6 more ..., Session | null>> & ... 12 more ... & Prefixer<...>, "viewer.">[TPath]' does not satisfy the constraint 'Procedure<any, any, any, any, any, any, any, any>'.ts(2344)`